c++/src/kj/compat/http-test.c++

```diff
@@ -6573,7 +6573,7 @@ KJ_TEST("Simple CONNECT Server works") {
              "\r\n"
              "hello"_kj).wait(waitScope);
 
-  expectEnd(*pipe.ends[1]);
+  expectEnd(*pipe.ends[1]).wait(waitScope);
```

Comment:
The fact that the tests were failing to await all these is kind of disturbing... and surprising that none of them fail when corrected.

Reply:
Yikes.